frontend/pages/schedule/ManageSchedulePage/components/ScheduleListWrapper/ScheduleListWrapper.tsx

```diff
@@ -13,19 +13,24 @@ import { ITeamScheduledQuery } from "interfaces/team_scheduled_query";
 import globalScheduledQueryActions from "redux/nodes/entities/global_scheduled_queries/actions";
 
 import TableContainer from "components/TableContainer";
-import { generateTableHeaders, generateDataSet } from "./ScheduleTableConfig";
+import {
+  generateInheritedQueriesTableHeaders,
+  generateTableHeaders,
+  generateDataSet,
+} from "./ScheduleTableConfig";
 // @ts-ignore
 import scheduleSvg from "../../../../../../assets/images/schedule.svg";
 
 const baseClass = "schedule-list-wrapper";
 const noScheduleClass = "no-schedule";
 
 interface IScheduleListWrapperProps {
-  onRemoveScheduledQueryClick: any;
-  onEditScheduledQueryClick: any;
+  onRemoveScheduledQueryClick?: any;
```

Comment:
Can you add types to these. It would require you to view the original function, but it would look something like this:
```typescript
(value: type) => otherType;
```

Or if it doesn't have parameter, then `() => type;`.